share/spice/spotify/spotify.js

```diff
@@ -0,0 +1,39 @@
+(function (env) {
+    'use strict';
+    env.ddg_spice_spotify = function(api_result){
+
+        var query = DDG.get_query();
+        query = query.replace(/spotify/, ''); //replace trigger words from query
+
+        // Validate the response (customize for your Spice)
+        if (!api_result || api_result.tracks.items.length === 0) {
+            return Spice.failed('spotify');
+        }
+
+        // Render the response
+        Spice.add({
+            id: 'spotify',
+
+            // Customize these properties
+            name: 'Spotify',
+            data: api_result.tracks.items,
+            meta: {
+                sourceName: 'Spotify',
+                sourceUrl: 'https://play.spotify.com/search/' + query.trim(),
+            },
+            normalize: function(item) {
+                return {
+                    title: item.name,
```

Comment:
@moollaza Would I need to use a custom footer to display the album name like [Soundcloud's](https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/share/spice/sound_cloud/footer.handlebars)? In that case, is there any CSS class to show ellipsis?